pillar/apps/ocw-production.sls

```diff
@@ -0,0 +1,45 @@
+ocw:
+  engines_conf:
+    database:
+      host: "10.100.0.7"
```

Comment:
It would be good to pull this in dynamically rather than hardcode IP addresses. Most likely using the Mine subsystem. You can refer to https://github.com/mitodl/salt-ops/blob/xpro_production_env/pillar/mongodb/init.sls#L7-L11 for an example of using the salt mine

Reply:
I've attempted to fix this in 8679d5c
Does it look OK?